datasets/glossis/vtkDockerfile

```diff
@@ -0,0 +1,21 @@
+FROM continuumio/miniconda3
+
+RUN \
+    apt-get update --fix-missing && \
+    apt-get install -y libglu1-mesa-dev && \
+  rm -rf /var/lib/apt/lists/*
+
+COPY vtk_environment.yml /tmp/environment.yml
+RUN conda env create -f /tmp/environment.yml
+
+RUN echo "source activate env" > /root/.bashrc
+ENV PATH=/opt/gcloud/google-cloud-sdk/bin:/opt/conda/envs/env/bin:$PATH 
+
+RUN mkdir /opt/gcloud /opt/datasets
+RUN (cd /opt/gcloud && curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-337.0.0-linux-x86_64.tar.gz && tar -xzf google-cloud-sdk-337.0.0-linux-x86_64.tar.gz)
```

Comment:
For how long will this version specific SDK link work? Might put it as a version number variable in the top of the file.